src/EhrgoHealth.WebService/Models/AllergyIntolerance.cs

```diff
@@ -0,0 +1,164 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using Hl7.Fhir.Model;
+using Hl7.Fhir.Rest;
+
+namespace EhrgoHealth.WebService.Models

```

Comment:
I'd personally not consider this a model, models in web land typically do not really have code, they are more representations of datastructures either in the database, or data passed into/from api calls